config/initializers/feature_toggle.rb

```diff
@@ -4,11 +4,11 @@ def self.saledate_collection_window_validation_enabled?
     Rails.env.production? || Rails.env.test? || Rails.env.staging?
   end
 
-  def self.startdate_two_week_validation_enabled?
-    Rails.env.production? || Rails.env.test? || Rails.env.staging?
+  def self.startdate_collection_window_validation_enabled?
+    !Rails.env.test?
   end
```

Comment:
This is a bad idea. Our tests should reflect the reality of the logic in production :)